fe/fe-core/src/main/java/org/apache/doris/httpv2/rest/LoadAction.java

```diff
@@ -257,4 +269,97 @@ private TNetworkAddress selectRedirectBackend(String clusterName) throws LoadExc
         }
         return new TNetworkAddress(backend.getHost(), backend.getHttpPort());
     }
+
+    // NOTE: This function can only be used for AuditlogPlugin stream load for now.
+    // AuditlogPlugin should be re-disigned carefully, and blow method focuses on
+    // temporarily addressing the users' needs for audit logs.
+    // So this function is not widely tested under general scenario
+    private boolean checkClusterToken(HttpServletRequest request) {
+        LOG.info("Checking cluser token, request {}", request.toString());
+        String authToken = request.getHeader("token");
+
+        if (Strings.isNullOrEmpty(authToken)) {
+            LOG.warn("Failed to check cluster token.");
```

Comment:
I think this is not a warn.
In normal case, if user submit load with wrong username/password, the `authToken` is always empty.
So I think we can remove this log

Reply:
Fixed